src/client/js/app.js

```diff
@@ -61,9 +67,9 @@ function performAction(e) {
                     baseURLWeatherbitDaily,
                     `key=${apiKeyWeatherbit}&lat=${lat}&lon=${lng}`
                 ).then((weatherDataDaily) => {
-                    const temperatureMax = weatherDataDaily.data[0].max_temp;
-                    const temperatureMin = weatherDataDaily.data[0].min_temp;
-                    const weatherDescriptionDaily = weatherDataDaily.data[0].weather.description;
+                    const temperatureMax = weatherDataDaily.data[12].max_temp;
+                    const temperatureMin = weatherDataDaily.data[12].min_temp;
+                    const weatherDescriptionDaily = weatherDataDaily.data[12].weather.description;
```

Comment:
Just curios, why it should be `12`? By the way I also think that `weatherDataDaily.data[12]` can be extracted into a variable for better readability. `dataItem` for instance.

Reply:
Yes, it may seem weird. 
In this project I use two different types of weather data:
1. If there are <= 7 days left before the trip I use Current Weather API.
2. If there are > 7 days left before the trip I use 16 Days Weather Forecast. It provides weather forecast for 16 days (daily) starting today. 
I don't need `weatherDataDaily.data[0]` because a trip can start only tomorrow. And I don't need data `weatherDataDaily.data` [from 1 to 8] because I use current weather for that case. 
So there are 7 days left `weatherDataDaily.data` [from 9 to 15], and `weatherDataDaily.data[12]` is kinda in a middle of a trip and seems to be a good choice.